alvr/server/cpp/platform/linux/EncodePipeline.cpp

```diff
@@ -78,26 +78,32 @@ std::unique_ptr<alvr::EncodePipeline> alvr::EncodePipeline::Create(Renderer *ren
         auto nvenc = std::make_unique<alvr::EncodePipelineNvEnc>(input_frame, vk_frame_ctx, width, height);
         Info("using NvEnc encoder");
         return nvenc;
-      } catch (...)
+      } catch (std::exception &e)
       {
-        Info("failed to create NvEnc encoder");
+        std::stringstream err;
+        err << "failed to create NvEnc encoder: " << e.what();
+        Info(err.str().c_str());
```

Comment:
`Info()`, `Warn()`, `Error()` etc support string interpolation like `printf()`. Same comment for two more `Info()` below.